plugin/trino-mysql/src/test/java/io/trino/plugin/mysql/TestMySqlTypeMapping.java

```diff
@@ -1005,6 +1005,17 @@ public void testUnsignedTypes()
                 .execute(getQueryRunner(), mysqlCreateAndInsert("tpch.mysql_test_unsigned"));
     }
 
+    @Test
+    public void testEnumTypes()
+    {
+        // make exception for mysql enum type as varchar, not char : https://trinodb.slack.com/archives/CGB0QHWSW/p1626249411412400
+        SqlDataTypeTest.create()
+                .addRoundTrip("ENUM", "small", createVarcharType(5), "small")
+                .addRoundTrip("ENUM", "medium", createVarcharType(6), "medium")
+                .addRoundTrip("ENUM", "x-large", createVarcharType(7), "x-large")
+                .execute(getQueryRunner(), mysqlCreateAndInsert("tpch.mysql_test_enum"));
```

Comment:
I believe this test doesn't work due to wrong syntax. You can refer to the latest `TestPostgreSqlTypeMapping#testEnum` which doesn't use `SqlDataTypeTest`.

Also, this test should be abled to run on IDE easily. I would recommend running the test before pushing the next commit. Please feel free to ping in Slack if you can't the test in your dev environment.